arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java

```diff
@@ -814,7 +814,7 @@ private void setCreate(CreateOobCall createOobCall) {
             BigInteger.ZERO,
             createOobCall.getCreatorNonce(),
             BigInteger.ZERO,
-            longToUnsignedBigInteger(EIP2681_MAX_NONCE));
+            EIP2681_MAX_NONCE);
```

Comment:
@letypequividelespoubelles Conversion unnecessary now because `EIP2681_MAX_NONCE` is now already a `BigInteger`.